stdlib/3/os/__init__.pyi

```diff
@@ -124,7 +124,10 @@ class stat_result:
     st_ctime = 0.0 # platform dependent (time of most recent metadata change
                    # on  Unix, or the time of creation on Windows)
 
-    def __init__(self, tuple) -> None: ...
+    # not documented
+    def __init__(self,
+                 tuple: Tuple[int, int, int, int, int, int, int, int, int, int]) \
```

Comment:
Weird to have variable length for a struct like this; anyway, `Tuple[int, ...]` it is.